src/client/gui/gui_cmd.cpp

```diff
@@ -206,8 +217,12 @@ void cmd::GuiCmd::update_about_menu()
 
     if (update_available(reply.update_info()))
     {
+        update_action.setIcon(QApplication::style()->standardIcon(QStyle::SP_MessageBoxInformation));
         update_action.setWhatsThis(QString::fromStdString(reply.update_info().url()));
         tray_icon_menu.insertAction(about_menu.menuAction(), &update_action);
+        tray_icon.showMessage("New Multipass update available",
+                              QString("Version %1 is availble. Click for more information.")
```

Comment:
```suggestion
                              QString("Version %1 is availble. Click here for more information.")
```
I know the "here" bit can't actually be a link, but "Click for…" kinda begs the question "where do I click again?".

Reply:
Ok, sure.

Reply:
Done.  And I also fixed a type in what I originally had :grin: